Modules/ScormAicc/scripts/basisAPI.js

```diff
@@ -291,6 +292,26 @@ function status4tree(i_sco,s_status,s_time){
 	}
 }
 
+// Find out if we are in a beforeunload event
+function isBeforeUnloadEvent() {
```

Comment:
For some reasons I don't understand, this does not seem to work for SingleSCO, at least not for the one I am testing with. `event` is not set, neither on the window nor on any of the frames. Did someone test this with a SingleSCO-WBT? Is this something specific for the WBT I have tested with or is this a general problem with SingleSCO?

Reply:
Please check https://github.com/ILIAS-eLearning/ILIAS/commit/f9bec0286ece536059a02b674b59699044723ece 
My solution uses sendBeacon and works with singleSCO- and multiSCO-WBTs. Still an issue: Start last visited SCO at relaunch of a multiSCO-WBT.